activate.ps1

```diff
@@ -11,6 +11,11 @@ if ($MyInvocation.CommandOrigin -eq 'runspace') {
     exit 1
 }
 
+function dev-tye() {
```

Comment:
Thought on just making the name “tye” here instead of dev-tye? Would that work? 

It more similarly matches what activate does with dotnet. We don’t call dev-dotnet.

Reply:
Personally, I didn't want it to conflict with my globally installed "released" version of tye. Given that I have to have "activated" it though, I could be OK switching to `tye` if people prefer.

Reply:
Yeah, I'd say conflict with what is installed if it works.

Reply:
So you'd prefer that it *does* conflict? I can live with that I suppose.